lib/dor/item.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'retries'
+require 'dor-fetcher'
+
+module Dor::Release
```

Comment:
should this live in `lib/dor/release/item` ?

Reply:
Currently it's https://github.com/sul-dlss/item-release/blob/master/lib/dor/item.rb.  The idea is to not make too many changes here.